install_executorch.py

```diff
@@ -221,15 +219,15 @@ def main(args):
     # If --pybind is not set explicitly for backends (e.g., --pybind xnnpack)
     # or is not turned off explicitly (--pybind off)
     # then install XNNPACK by default.
-    if EXECUTORCH_BUILD_PYBIND == "":
-        EXECUTORCH_BUILD_PYBIND = "ON"
-        CMAKE_ARGS += " -DEXECUTORCH_BUILD_XNNPACK=ON"
+    if not has_pybindings:
+        has_pybindings = True
+        cmake_args.append("-DEXECUTORCH_BUILD_XNNPACK=ON")
```

Comment:
This logic.. I don't know what's going on..

Reply:
out of the scope of this PR though.

Reply:
before this change, there are three paths:

1. --pybindings off # turns off pybind altogether
2. --pybindings coreml xnnpack # turns on explicitly coreml and xnnpack
3. no pybindings options # turns on xnnpack by default

How do you deal with three situations with only has_pybindings=True | False?

Reply:
@mergennachin yes I didn't account for just off, made the changes now and included that in the test case in the description